twake/frontend/src/app/services/languages/locale/en.js

```diff
@@ -627,7 +626,8 @@ export default {
   'scenes.apps.account.languages.title': 'Language',
   'scenes.apps.account.languages.text': 'Change Twake language.',
   'scenes.apps.account.notifications.title': 'Notifications',
-  'scenes.apps.account.notifications.text': 'You can choose which notifications you want to receive here.',
+  'scenes.apps.account.notifications.text':
```

Comment:
You should not change things which are not related to the current work. This is the same for some changes below